recipes/gm2calc/all/conanfile.py

```diff
@@ -1,4 +1,5 @@
 import os
+import conan
```

Comment:
```suggestion
```

Reply:
Sorry, I cannot apply these suggestions, because the PR has been merged and is closed. Shall I open another PR to include these?